backend/services/theme_service.py

```
@@ -638,50 +536,35 @@ async def generate_themed_image_bytes(
                 analyzed_skin_color = "warm natural skin tone with consistent complexion"
             
             # Create smart person-background separation mask 
-            mask_bytes = self._create_smart_person_background_mask(base_image_bytes, image_width, image_height)
+            mask_bytes = self._create_simple_face_mask(base_image_bytes, image_width, image_height)
```

Ruff (0.12.2)

Local variable `mask_bytes` is assigned to but never used

Remove assignment to unused variable `mask_bytes`

(F841)